clients/AMQP/rhea.js

```diff
@@ -0,0 +1,10 @@
+// Binding for rheajs client
+module.exports = {
+  mapping: {
+    host: 'host',
+    password: 'password',
+    node: 'node',
```

Comment:
@pacostas what is in node when provided by the service bindings?  I don't see it being used in the messaging starter and just want to be sure it is a parameter that should be included in the options.

Reply:
Yes, that is a mistake,ty!